treeopt/visualize.py

```diff
@@ -0,0 +1,370 @@
+import matplotlib.pyplot as plt
```

Comment:
In general it would be nice to have some examples on how to use the visualization classes, either in the docstrings or in the examples or both. This does not have to be fixed as part of this PR, maybe a seperate issue.